lib/active_force/association/eager_load_builder_for_nested_includes.rb

```diff
@@ -0,0 +1,84 @@
+module ActiveForce
+
+  module Association
+    class InvalidAssociationError < StandardError; end
```

Comment:
Why is this defined here and [here](https://github.com/Beyond-Finance/active_force/pull/62/files#diff-84dd10b7e5c6c9d74b0e0881d7a87fd795d9ce3c9b2210b9b5f1f56d5a204387R7)?